test/integration/command_test.exs

```diff
@@ -13,6 +13,11 @@ defmodule Integration.CommandTest do
     assert response["data"]["response"] == "req-opt response"
   end
 
+  test "running a command with a bad template", %{user: user} do
+    response = send_message(user, "@bot: operable:bad-template")
+    assert response["data"]["response"] == "@vanstee Whoops! An error occurred. There was an error rendering the template 'badtemplate' for the adapter 'test': %Protocol.UndefinedError{description: nil, protocol: String.Chars, value: %{\"bad\" => %{\"foo\" => \"bar\"}}}"
```

Comment:
That's actually why I left it out the first time. I wasn't sure how useful it would be. We get the `Protocol.UndefinedError` when we try to serialize a map. That's actually the only thing I could find that causes us to crash when we render a template. I could match on `Protocol.Undefined` and return something like, "Error serializing value: #{value}". And then have a default case that just inspects the error. Thoughts?